skill_tagging/handlers.py

```diff
@@ -10,9 +11,10 @@ def listen_for_xblock_skill_verified(**kwargs):
     """
     Publish openedx-event XBLOCK_SKILL_VERIFIED signal onto the event bus.
     """
+    topic = getattr(settings, "EVENT_BUS_XBLOCK_VERIFICATION_TOPIC", "learning-xblock-skill-verified")
```

Comment:
Same suggestion as given here https://github.com/openedx/edx-platform/pull/32449#discussion_r1230716419

Reply:
As mentioned in edx-platform PR, it is combined by underlying implementation.